src/index.html

```diff
@@ -5,9 +5,61 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet">
+    <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img
+        src="images/imac.jpeg"
+        alt="Apple iMac 27&quot;"
+        class="card__img">
+
+      <p class="card__name">
+          APPLE A1419 iMac 27" Retina <br>
+          5K Monoblock (MNED2UA/A)
+      </p>
+
+      <p class="card__code">
+        Product code: 195434
+      </p>
+
+      <div class="card__votes">
```

Comment:
```suggestion
      <div class="card__reviews">
```
I think reviews sounds better 😄

Reply:
updated in solution3